conbench/entities/history.py

```diff
@@ -302,30 +306,40 @@ def set_z_scores(benchmark_results: List[BenchmarkResult], baseline_commit: Comm
         )
 
 
-def _query_distribution_stats_by_run_id(
-    run_id: str,
+def _query_and_calculate_distribution_stats(
+    contender_run_id: str,
     baseline_commit: Commit,
     case_id: Optional[str],
     context_id: Optional[str],
 ) -> Dict[Tuple[str, str], Tuple[Optional[float], Optional[float]]]:
-    """Given a run id and baseline commit, return stats of the distribution of
-    BenchmarkResults matching the run's hardware, in the baseline_commit's ancestry, by
-    case/context. Returns a dict that looks like
+    """Query and calculate rolling stats of the distribution of all BenchmarkResults
+    that:
```

Comment:
@jgehrcke Let me know if this name and docstring clarifies your questions better.